viewing_party/party.py

```diff
@@ -1,23 +1,128 @@
-# ------------- WAVE 1 --------------------
+#------------------------------------------------
+# THESE ARE DEFINED HELPER FUNCTIONS FOR WAVE 3+#
+#------------------------------------------------
+
+import collections
+
+def generate_user_dict(user_info,dict_key): #makes the dict of user watched movies
+    a_dict={}
+    for entry in user_info[dict_key]:
+        a_dict[entry["title"]]=entry
+    return a_dict
+def generate_friends_dict(user_info): #makes the dict of friend watched movies

```

Comment:
This is a minor thing, but for readability I recommend adding some whitespace between functions.
```suggestion
    return a_dict
    
def generate_friends_dict(user_info): #makes the dict of friend watched movies

```